notebooks/official/pipelines/control_flow_kfp.ipynb

```diff
@@ -74,17 +76,16 @@
       "source": [
         "### Objective\n",
         "\n",
-        "In this tutorial, you learn how to use the KFP SDK to build pipelines that use loops and conditionals, including nested examples.\n",
+        "In this tutorial, you learn how to use the KFP SDK to build pipelines that uses loops and conditionals, including nested examples.\n",
```

Comment:
parsing is not great. Suggested change: (I'm assuming "uses" agrees with "the KFP SDK" and not "pipelines, which is why you changed it from "use". But, it's very awkward as written above.)

- "In this tutorial, you learn how to use the KFP SDK, which uses loops and conditional including nested examples, to build pipelines."